src/NuGet.Clients/NuGet.VisualStudio.Implementation/Extensibility/VsPathContextProvider.cs

```diff
@@ -169,7 +171,7 @@ private async Task<NuGetProject> CreateNuGetProjectAsync(string projectUniqueNam
             await NuGetUIThreadHelper.JoinableTaskFactory.SwitchToMainThreadAsync();
 
             var dte = await _asyncServiceprovider.GetDTEAsync();
-            var supportedProjects = dte.Solution.Projects.Cast<EnvDTE.Project>();
+            var supportedProjects = GetProjectsInSolution(dte);
```

Comment:
See rule 10 in https://github.com/NuGet/NuGet.Client/blob/dev/docs/coding-guidelines.md#c-coding-style.